docs/environment.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
 - nbformat
 - jupyter_client
 - ipython
-- sphinx>=1.3.6
+- sphinx>=1.3.6,!=1.5.0,!=1.5.1
```

Comment:
@takluyver I would say hopeful of 1.5.2 but not 100% confidence. @michaelpacer You could make this 1.3.6 <= sphinx < 1.5.0 if you wish. Or leave as is. I don't have a strong preference either way.